src/Types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export type Todo = {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  user?: User;
```

Comment:
If the userId is always required, why user should be optional?

Reply:
I did this because the function to get *user* can return undefined(find method).